src/Common/src/CoreLib/System/ReadOnlySpan.cs

```diff
@@ -139,30 +136,6 @@ internal ReadOnlySpan(ref T ptr, int length)
             return ref _pointer.Value;
         }
 
-        /// <summary>
-        /// The number of items in the read-only span.
-        /// </summary>
-        public int Length
-        {
-            [NonVersionable]
```

Comment:
Why did we have some properties marked as NonVersionable before?

cc @jkotas

Reply:
Do we need ` using System.Runtime.Versioning;` in this file anymore?

Reply:
This is important for R2R code quality (the attribute allows cross-module inlining for these properties). We should keep the attributes for the Fast version.